server/utils/sendEmail.js

```diff
@@ -0,0 +1,22 @@
+const sgMail = require("@sendgrid/mail");
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const send = async (message) => {
```

Comment:
It is better to modify the send() to send(to, subject, message). And put the  from: process.env.FROM_EMAIL inside the send() so that the others can easily reuse the send().

Reply:
that is a good suggestion but the send method accepts one argument in which i am passing the entire email details for different usages